kernel/public/ecs-scenes/0.107.signed-fetch/game.ts

```diff
@@ -0,0 +1,27 @@
+import { signedFetch } from '@decentraland/SignedFetch'
+
+const cube = new Entity()
+
+cube.addComponentOrReplace(
+  new Transform({
+    position: new Vector3(5, 1, 5)
+  })
+)
+
+const text = new TextShape('Wait for it')
+text.billboard = true
+text.isPickable = true
+cube.addComponentOrReplace(text)
+
+engine.addEntity(cube)
+
+executeTask(async () => {
+  const questsResponse = await signedFetch('https://quests-api.decentraland.io/quests', { responseBodyType: 'json' })
```

Comment:
We are able to make this particular request without the client because this is not obfuscated.

I didn't test the client here since it is an external dependency.